config/rollup.js

```diff
@@ -0,0 +1,90 @@
+import babelPlugin from 'rollup-plugin-babel'
+import commonjsPlugin from 'rollup-plugin-commonjs'
+import fs from 'fs'
+import { resolve } from 'path'
+import resolvePlugin from 'rollup-plugin-node-resolve'
+import uglify from 'rollup-plugin-uglify'
+
+const root = resolve(__dirname, '..')
+const nodeModules = resolve(root, 'node_modules')
+
+const makeBundle = (name, options = {}, minify = false) => {
```

Comment:
I recommend to comment each statement because we won't remember why we have put some options in 6 month.

Reply:
I'll let @nlepage explain what he has done to make it work with immutadot-lodash :P